### DISCUSSION TITLE
Start all new update changelogs with:

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1079] [Ideas] [answered]
Typing monkeys of the internet a new update is here!

Something super similar was used for the V1.5.8 Release Changelog , It would be fun to see it come to all release changelogs!

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Yeah it would be cool for the update logs to be a little more enthusiastic and fun. Kinda like the name "Monkeytype". Perhaps over time inside jokes could be developed through the update changelogs. Or foreshadowing to the next update or major update (such as multiplayer, friend list, new features, or maybe a major overhaul).

**Reply by ObiKeahloa** [reply]
Yeah true , was surprised when I saw that in the 1.5.8 Changelog in GitHub.

Would love to see the changelogs become more and more secretive and cheeky!

Edit: Wrote GitHub as GutHub ;)

**Reply by frogfrogfrogfrogfrog** [reply]
I wonder why this was chosen as the answer

**Reply by Miodec** [reply]
> I wonder why this was chosen as the answer

I may have forgot to actually answer 😳 

Anyway, the answer is I try to give the bigger updates more flavour where I can, but keep the small bug fixes relatively boring because well - its just bug fixes.

It would be hard to come up with something original for every single small update, and keeping the opening line the same is not something i like the sound of i think

**Reply by ObiKeahloa** [reply]
Well yeah makes sense

**Reply by frogfrogfrogfrogfrog** [reply]
yeeap
